### DISCUSSION TITLE
How to copy content from a Group shared by an external organisation

### DISCUSSION THREAD

**Discussion by NRW-INSPIRE** [EsriPS/arcgis-assistant-feedback#65] [Q&A]
A third party organisation has produced content for us, including an Experience Builder app. We would like to copy this content over to our own ArcGIS Online account.

The old ArcGIS Online Assistant is able to see Groups, but is only capable of copying some types of data like features and instant apps - but not Experience Builder apps.

The new ArcGIS Assistant doesn't seem to have the ability to view content saved in groups. Am i missing something?

**Comment by jpeterson** [comment]
Hey @NRW-INSPIRE, sorry for the delay in getting back to you. ArcGIS Assistant does not currently support discovering items via groups – only folders.
